custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -672,17 +678,18 @@ def run_test_predict(self):
                 df.to_csv(__tempfile.name, index=False, line_terminator="\r\n")
             self.options.input = __tempfile.name
 
-        if self.target_type == TargetType.TRANSFORM:
-            CMRunTests(
-                self.options, self.run_mode, self.target_type, self.schema_validator
-            ).check_transform_server(__target_temp)
-        else:
-            try:
+        try:
+            if self.target_type == TargetType.TRANSFORM:
+                CMRunTests(
+                    self.options, self.target_type, self.schema_validator
+                ).check_transform_server(__target_temp)
+            else:
+
                 CMRunTests(
-                    self.options, self.run_mode, self.target_type, self.schema_validator
+                    self.options, self.target_type, self.schema_validator
                 ).check_prediction_side_effects()
-            except DrumPredException as e:
-                self.logger.warning(e)
+        except DrumPredException as e:
+            self.logger.warning(e)
```

Comment:
Out of curiosity, why are you removing self.run_mode from `CMRunTests` above? Is it an unused param?

Reply:
Yes, it was unused.